airgun/browser.py

```diff
@@ -196,10 +196,32 @@ def _get_selenium_browser(self):
             self._webdriver = webdriver.PhantomJS(
                 service_args=['--ignore-ssl-errors=true'])
         elif self.browser == 'remote':
-            capabilities = vars(settings.webdriver_desired_capabilities)
+            browserName = settings.webdriver_desired_capabilities.browserName
```

Comment:
Do you need to introduce new variable and setting here? Can't you re-use `self.browser` (defaults to `settings.selenium.webdriver`)?

webdriver capability `browserName` will be set by `webdriver.DesiredCapabilities.<provider>.copy()` anyway.

Reply:
+1 to what @mirzal said.

Reply:
Because there are different browsers used for remote, in my opinion, remote should not be a value for 'settings.selenium.webdriver', it should be a provider for 'settings.selenium.browser' 
provider = ('selenium', 'docker', 'saucelabs', 'remote')
browser = ('chrome', 'firefox', 'ie', 'edge', 'phantomjs')

Reply:
> Because there are different browsers used for remote, in my opinion, remote should not be a value for `settings.selenium.webdriver`, it should be a provider for `settings.selenium.browser`:
> provider = ('selenium', 'docker', 'saucelabs', 'remote')
> browser = ('chrome', 'firefox', 'ie', 'edge', 'phantomjs')

Fully agreed. In fact, my comment made sense only if you assume such situation - of course you can't use `self.browser` in these ifs now, because `self.browser` is `remote`.

Is moving `remote` to `provider` something you can work on as part of this PR?
As you have shown, `remote` hardly works right now at all, so I'm not worried too much about backwards compatibility here.